frontend/src/components/maps/leaflet/mapUtils.tsx

```diff
@@ -154,39 +244,70 @@ export const pointToLayer = (feature: ICluster, latlng: LatLngExpression): Layer
 /**
  * Get an icon type for the specified cluster property details (type, draft, erp, spp etc)
  */
-export const getMarkerIcon = (feature: ICluster) => {
+export const getMarkerIcon = (feature: ICluster, selected?: boolean) => {
   const { propertyTypeId, projectWorkflow, classificationId } = feature?.properties;
-  if (projectWorkflow === Workflows.ERP) {
-    switch (propertyTypeId) {
-      case PropertyTypes.SUBDIVISION:
-        return subdivisionErpIcon;
-      case PropertyTypes.BUILDING:
-        return buildingErpIcon;
-      default:
-        return landErpIcon;
-    }
-  } else if (
-    classificationId === Classifications.SurplusActive ||
-    classificationId === Classifications.SurplusEncumbered
-  ) {
-    switch (propertyTypeId) {
-      case PropertyTypes.BUILDING:
-        return buildingSppIcon;
-      case PropertyTypes.SUBDIVISION:
-        return subdivisionSppIcon;
-      default:
-        return landSppIcon;
-    }
-  } else if (propertyTypeId === PropertyTypes.PARCEL) {
-    return parcelIcon;
-  } else if (propertyTypeId === PropertyTypes.SUBDIVISION) {
-    return subdivisionIcon;
-  } else if (propertyTypeId === PropertyTypes.DRAFT_PARCEL) {
+  if (propertyTypeId === PropertyTypes.DRAFT_PARCEL) {
     return draftParcelIcon;
   } else if (propertyTypeId === PropertyTypes.DRAFT_BUILDING) {
     return draftBuildingIcon;
+  } else if (selected) {
```

Comment:
I wasn't sure if it would be better to do one global check here, or check in the return statement for each type